src/cli/README.md

```diff
@@ -210,6 +212,64 @@ Set the Thread Child Timeout value.
 Done
 ```
 
+### commissioner start \<psdk\> \<provisioningUrl\>
+
+Start the Commissioner role.
+
+* pskd: Pre-Shared Key for the Joiner.
+* provisioningUrl: Provisioning URL for the Joiner (optional).
+
+This command will cause the device to send LEAD_PET and LEAD_KA messages.
+
+```bash
+> commissioner start ot-psk
+Done
+```
+
+### commissioner stop
+
+Stop the Commissioner role.
+
+This command will cuase the device to send LEAD_KA messages.
+
+```bash
+> commissioner stop
+Done
+```
+
+### commissioner energy \<mask\> \<count\> \<period\> \<scanDuration\> \<destination\>
+
+Send a MGMT_ED_SCAN message.
+
+* mask: Channels to perform IEEE 802.15.4 ED Scans.
+* count: Number of IEEE 802.15.4 ED Scans per channel.
+* period: Period between successive IEEE 802.15.4 ED Scans.
+* destination: IPv6 destination for the message (may be multicast).
+
+Energy reports are printed as MGMT_ED_REPORT messages are received.
+
+```bash
+> commissioner energy 0x00050000 2 32 1000 fdde:ad00:beef:0:0:ff:fe00:c00
+Done
+Energy: 00050000 0 0 0 0
```

Comment:
May want to explain output.
What does final 0 mean, for example, destination = 0?  
Is it the address of the node that collected the reading?